shell/browser/api/electron_api_web_contents.cc

```diff
@@ -386,11 +386,12 @@ base::IDMap<WebContents*>& GetAllWebContents() {
   return *s_all_web_contents;
 }
 
-// Called when CapturePage is done.
 void OnCapturePageDone(gin_helper::Promise<gfx::Image> promise,
+                       base::ScopedClosureRunner capture_handle,
                        const SkBitmap& bitmap) {
   // Hack to enable transparency in captured image
   promise.Resolve(gfx::Image::CreateFrom1xBitmap(bitmap));
+  capture_handle.RunAndReset();
```

Comment:
```suggestion
  // Hack to enable transparency in captured image
  promise.Resolve(gfx::Image::CreateFrom1xBitmap(bitmap));
  
  capture_handle.RunAndReset();
```

_(nit)_ suggest a paragraph break between the two lines to make it clear that the transparency hack mentioned in the comment does not depend on capture_handle